invoices_app/forms.py

```diff
@@ -31,22 +32,54 @@ class Meta:
             'invoice_type': forms.Select(attrs={'class': 'custom-select'}),
             'currency': forms.Select(attrs={'class': 'custom-select'}),
             'po_number': forms.NumberInput(attrs={'class': 'form-control', 'placeholder': 'Purchase Order'}),
-            'invoice_file': forms.FileInput(attrs={'accept': 'application/pdf'}),
-            'po_file': forms.FileInput(attrs={'accept': 'application/pdf'}),
+            'invoice_file': forms.FileInput(
+                attrs={
+                    'type': 'file',
+                    'class': 'custom-file-input',
+                    'name': 'invoice_file',
+                    'id': 'inputGroupFile01',
+                    'aria-describedby': 'inputGroupFileAddon01',
+                    'accept': 'application/pdf',
+                }
+            ),
+            'po_file': forms.FileInput(
+                attrs={
+                    'type': 'file',
+                    'class': 'custom-file-input',
+                    'name': 'invoice_file',
+                    'id': 'inputGroupFile02',
+                    'aria-describedby': 'inputGroupFileAddon01',
+                    'accept': 'application/pdf',
+                }
+            ),
             'vat': forms.NumberInput(attrs={'class': 'form-control', 'placeholder': 'VAT'}),
             'total_amount': forms.NumberInput(attrs={'class': 'form-control', 'placeholder': 'Total'}),
             'net_amount': forms.NumberInput(attrs={'class': 'form-control',  'placeholder': 'Net Amount'}),
             'invoice_number': forms.NumberInput(attrs={'class': 'form-control', 'placeholder': 'Invoice Number'}),
         }
 
-    def is_valid(self):
-        valid_from_super = super().is_valid()
 
-        if not self.fields.get('invoice_file'):
+    def is_valid(self):
+        super().is_valid()
+        if not self.files.get('invoice_file'):
+            self.add_error('invoice_file', 'No Invoice File')
             return False
 
-        return is_file_valid(
-            self,
-            valid_from_super,
-            self.fields['invoice_file']
-        )
+        file_fields = list(self.files.keys())
+        files_valids = {}
+
+        if file_fields:
```

Comment:
Creo que este if no es necesario. Si file_fields es vacío, va a ser una lista vacía. Cuando lo iteres con un for va intentar recorrerlo, y va a seguir, porque la lista está vacía.